src/app/pages/components/components.html

```diff
@@ -1,4 +1,28 @@
-<div *ngFor="let component of componentItems">
-  <img src="../../../assets/img/components/{{component.src}}.svg" [alt]="component.name" />
-  {{component.name}}
+<div class="docs-components-header-bar">Components</div>
+<div class="docs-components-component-list">
+  <div class="docs-components-header-section">
+    <h1>Directory of Design Components</h1>
+    <p>
+      Set of high-quality UI components built with Angular 2 and
+      TypeScript, following the Material Design spec
+    </p>
+  </div>
+  <div [class.docs-components-grid-view]="gridMode" class="docs-components-view">
+    <button md-fab color="primary" (click)="switchViewMode()" class="docs-components-grid-fab">
```

Comment:
This should actually be a button-toggle (I noticed this when we were looking at the mocks yesterday, there hasn't been time to update them yet)